ddp-workspace/projects/ddp-mbc/src/app/components/more-details/more-details.component.ts

```diff
@@ -139,19 +142,20 @@ export class MoreDetailsComponent implements OnInit {
     public infoEmailHref: string;
     public phoneHref: string;
     public countMeInUrl: string;
-    public facebookUrl: string;
+    public facebookUrl$: Observable<string>;
     public twitterUrl: string;
     public instagramUrl: string;
 
-    constructor(@Inject('toolkit.toolkitConfig') private toolkitConfiguration: ToolkitConfigurationService) { }
+    constructor(@Inject('toolkit.toolkitConfig') private toolkitConfiguration: ToolkitConfigurationService,
+                private ngxTranslate: NGXTranslateService) { }
 
     public ngOnInit(): void {
         this.infoEmail = this.toolkitConfiguration.infoEmail;
         this.infoEmailHref = `mailto:${this.toolkitConfiguration.infoEmail}`;
         this.phone = this.toolkitConfiguration.phone;
         this.phoneHref = `tel:${this.toolkitConfiguration.phone}`;
         this.countMeInUrl = this.toolkitConfiguration.countMeInUrl;
-        this.facebookUrl = `https://www.facebook.com/${this.toolkitConfiguration.facebookGroupId}`;
+        this.facebookUrl$ = this.ngxTranslate.getTranslation('Toolkit.Footer.FacebookLink');
```

Comment:
We were getting the Facebook link from the configuration, but now that single value is not going to work for us. so getting it from translation files.